packages/flutter_tools/test/general.shard/ios/ios_device_start_nonprebuilt_test.dart

```diff
@@ -43,17 +43,16 @@ const List<String> kRunReleaseArgs = <String>[
   '-configuration',
   'Release',
   '-quiet',
+  '-sdk',
+  'iphoneos',
   '-workspace',
   'Runner.xcworkspace',
   '-scheme',
   'Runner',
   'BUILD_DIR=/build/ios',
-  '-sdk',
-  'iphoneos',
   'ONLY_ACTIVE_ARCH=YES',
   'ARCHS=arm64',
-  'FLUTTER_SUPPRESS_ANALYTICS=true',
```

Comment:
We actually want the analytics now that each re-entrant build corresponds to a single flutter invocation